modules/installing-ocp-agent-ibm-z-kvm.adoc

```diff
@@ -14,11 +14,6 @@ endif::[]
 Use the following procedure to manually add {ibm-z-name} agents with {op-system-base} KVM.
 Only use this procedure for {ibm-z-name} clusters with {op-system-base} KVM.
 
-[NOTE]
```

Comment:
@krishnanshruthi I just noticed that this level 1 heading in line 12 still says `{ibm-z-name}` 
Can you please change that to: `Adding {ibm-z-title} agents with {op-system-base} KVM`

Background is that trademarks cause problems in level 1 headings on the customer portal.